source/_posts/2024-06-05-release-20246.markdown

```diff
@@ -0,0 +1,734 @@
+  sequences of actions in [parallel].
+- Sensors can be [grouped] to calculate, for example, their sum or mean value.
+  Thanks to [@CoRfr], it is now possible to calculate the [standard deviation]
+  of a group of sensors. Cool!
+
+[@CoRfr]: https://github.com/CoRfr
+[@frenck]: https://github.com/frenck
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[@surfingbytes]: https://github.com/surfingbytes
+[@Troon]: https://github.com/Troon
+[grouped]: /integrations/group
+[parallel]: /docs/scripts/#parallelizing-actions
+[Run sequence of actions]: /docs/scripts/#grouping-actions
+[standard deviation]: https://en.wikipedia.org/wiki/Standard_deviation
+
+### Disabling expiration of login tokens
+
+A few releases ago, we introduced the automatic expiration of logins. If a login
+session wasn't used for 90 days, it would expire and be automatically cleaned up.
+
+Some of you have expressed that you would like to disable this feature for
+various reasons. For example, because of a secondary holiday home, one doesn’t
+regularly visit, or because of relatives who can only come over a few times
+a year, we added a new option to the user profile settings to disable
+the expiration of specific login tokens.
+
+<img class="no-shadow" src='/images/blog/2024-06/disabling-expiration-login-tokens.png' alt='Screenshot showing the option to disable the expiration of tokens in the profile page.'>
+
+You can find these token in your profile settings, under the **Security** tab.
+
+{% my profile badge %}
+
+### Tag entities!
+
+Home Assistant has support for tags for a long time. The tags feature allowed
+NFC tags or QR codes to be used with Home Assistant, for example, to
+trigger {% term automations %}.
+
+However, tags were not real entities and, thus, not always as logical to use.
+With this release, [@gjohansson-ST] changed this!
+
+Tags are now normal {% term entities %} and can be used in {% term automations %},
+{% term scripts %}, {% term templates %}, and even added to
+your [dashboards](/dashboards)!
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+
+### Collapsable sections for blueprints!
+
+[@karwosts] makes an appearance again this release with another of his QoL 
+(Quality of Live) improvements. He added the ability to add collapsible sections
+to {% term blueprints %}. This allows blueprint creators to group configuration
+options together or hide advanced or less common options by default.
+
+<p class='img'>
+    <img src="/images/blog/2024-06/blueprint-input-sections.gif" alt="Screenrecording showing sections of Blueprints inputs being collapsed/uncollapsed."/>
+    Each section can be collapsed or uncollapsed by clicking on the section header.
+</p>
+
+Thanks [@karwosts]!
+
+Read more about blueprint input sections [in our documentation](/docs/blueprint/schema/#blueprint-input-sections).
+
+Looking for an interesting blueprint to get started? Check out the 
+[Blueprints Exchange] on our community forum! There are great automation
```

Comment:
There seems to be a double space here when looking at the release notes, maybe because of the added space at the end?
```suggestion
Looking for an interesting blueprint to get started? Check out the
[Blueprints Exchange] on our community forum! There are great automation
```